Mention supported network path, default values for optional fields and link to custom API source creation (#21)
elastic/enterprise-search-network-drives-connector: README.md

Collect the information that is required to connect to your network drives:

- The domain name where the network drive is present.

> - The network drives path the connector will crawl to fetch files. This is the name of the folder shared via SMB, followed by the path relative to that shared folder. For example: if the name of the of SMB shared folder is `Users` and the folder to be fetched is `folder1`, then the path is `Users/folder1`.

Author: The connector supports SMB1 and SMB2. Adding this in README

Collect the information that is required to connect to your network drives:

- The domain name where the network drive is present.

> - The network drives path the connector will crawl to fetch files. It should be the name of the folder shared via SMB followed by the path relative to that shared folder
> 
>   (For Example: If `Users` is the name of SMB shared folder and folder to be fetched is `folder1`, then the path would be mentioned as `Users/folder1`)

Reviewer: ```suggestion
- The network drives path the connector will crawl to fetch files. This is the name of the folder shared via SMB, followed by the path relative to that shared folder. For example: if the name of the of SMB shared folder is `Users` and the folder to be fetched is `folder1`, then the path is `Users/folder1`).
```

Reviewer: Minor language tidy up

Later, you will [configure the connector](#configure-the-connector) with these values.

> ℹ️ The connector uses [pysmb](https://pysmb.readthedocs.io/en/latest/) module for connecting to your Network Drives. pysmb supports SMB1 and SMB2.

Reviewer: ```suggestion
ℹ️ The connector uses the [pysmb](https://pysmb.readthedocs.io/en/latest/) module for connecting to your Network Drives. `pysmb` supports SMB1 and SMB2.
```

Create a content source within Kibana:

1. Navigate to **Enterprise Search** → **Workplace Search** → **Sources** → **Add an organization content source** → **Custom API Source**.
2. Name your Content Source, (e.g. Network Drives Connector).

> For more details please refer [Elastic Documentation for creating a custom content source](https://www.elastic.co/guide/en/workplace-search/current/workplace-search-custom-api-sources.html#create-custom-source).

Reviewer: ```suggestion
For more details see [Elastic Documentation for creating a custom content source](https://www.elastic.co/guide/en/workplace-search/current/workplace-search-custom-api-sources.html#create-custom-source).
```

Whether the connector should sync [document-level permissions (DLP)](#use-document-level-permissions-dlp) from network drives.

> By default, it is set to `Yes` i.e. by default, the connector will try to sync document-level permissions.

Reviewer: ```suggestion
Set to `Yes` by default i.e. the connector will try to sync document-level permissions.
```
